skimage/filters/thresholding.py

```diff
@@ -142,7 +142,7 @@ def wrapper(im):
                     methods=methods, verbose=verbose)
 
 
-def threshold_local(image, block_size, method='gaussian', offset=0,
+def threshold_local(image, block_size=2, method='gaussian', offset=0,
```

Comment:
The description of block_size in the docs below asks for an odd value, so 2 does not seem like a good choice. Either we need to update the description or choose a different value here and also enforce that the input is odd using a check.

Reply:
I changed the default block_size from 2 to 3.